src/test/kotlin/water/blackjack/model/CardsDeckTest.kt

```diff
@@ -0,0 +1,43 @@
+package water.blackjack.model
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import water.blackjack.exception.BlackJackException
+import water.blackjack.exception.ExceptionMessages
+
+class CardsDeckTest {
+    private val totalCardCount = 52
+
+    @Test
+    fun `52장의 카드를 생성한다`() {
+        val deck = CardsDeck()
+        deck.offerCards(totalCardCount)
+
+        val exception = assertThrows<BlackJackException> {
+            deck.offerCards(1)
+        }
+        assertEquals(exception.message, ExceptionMessages.OUT_OF_CARD_MESSAGE)
+    }
+
+    @Test
+    fun `요청한 카드 수 만큼 서로 다른 카드를 반환한다`() {
+        val deck = CardsDeck()
+        val requiredCount = 2
+        val offeredCards = deck.offerCards(requiredCount)
+        assertEquals(requiredCount, offeredCards.size)
```

Comment:
```suggestion
        //given
        val deck = CardsDeck()
        val requiredCount = 2
        
        //when
        val offeredCards = deck.offerCards(requiredCount)
        
        //then
        assertEquals(requiredCount, offeredCards.size)
```
given-when-then으로 나누시면 가독성이 보다 올라갈 거 같아요!